posts/benchmarks/_posts/2011-9-1-animation.html

```diff
@@ -93,6 +96,16 @@
 
     var angle = 0;
     function animate() {
+      var time = new Date(),
+          currentFps = (Math.round(1000 / (time - prevTime))),
```

Comment:
Everything look great, thanks. Minor note: no need for parenthesis here, around `Math.round(...)`